### DISCUSSION TITLE
Output pins on multiple side

### DISCUSSION THREAD

**Discussion by tpagarani** [lnis-uofu/OpenFPGA#1012] [Q&A]
Does OpenFPGA support tile definitions with same pin duplicated on more than one side?  While VPR XML allows to define it, ran into this assertion failure OpenFPGA-v1.2.0/openfpga/src/utils/openfpga_physical_tile_utils.cpp:44

**Comment by tangxifan** [comment]
Hi Tarachand,
  If it is an output pin, it is allowed to appear on different sides of a tile. We just do not allow to define duplicated pins in <pinlocation> of the XML. You should be able to define pins with different names at pb_type level, and wire them internally. If you do think this is a feature you need urgently, can you provide a simple test case, we can then talk offline about the technical support.
